src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -276,7 +277,7 @@ private void filmSuchen1(final String strUrlFeed, final String thema, final bool
         filmSuchen2(url, thema, titel, d, datum, zeit, urlSendung);
       }
       if (!Config.getStop() && weiter
-          && (CrawlerTool.loadLongMax() || thema.equalsIgnoreCase("alpha-Centauri"))) {
+              && (CrawlerTool.loadLongMax() || thema.equalsIgnoreCase("alpha-Centauri"))) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'THEMA_ALPHA_CENTAURI' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)